app/controllers/manage_metadata_controller.rb

```diff
@@ -24,25 +24,37 @@ def set_variable_information
     # if the variable is not found, try again because CMR might be a little slow to index if it is a newly published record
     attempts = 0
     while attempts < 20
-      variables_search_response = cmr_client.get_variables(concept_id: @concept_id)
+      variables_search_response = cmr_client.get_variables(concept_id: @concept_id, all_revisions: true)
 
       variable_data = if variables_search_response.success?
-                        variables_search_response.body['items'].first
+                        variables_search_response.body.fetch('items', [])
                       else
-                        {}
+                        [{}]
                       end
+      variable_data.sort! { |a, b| b['meta']['revision-id'] <=> a['meta']['revision-id'] }
 
-      break if !variable_data.nil? && variable_data.fetch('meta', {})['concept-id'] == @concept_id
+      # break if revisions is wrong
```

Comment:
is this a comment left in or something you need to implement?

Reply:
left in and now in a weird spot after stuff got moved around lol